WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/UIApplication+AppAvailability.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+enum AppScheme: String {
+    case wordpress = "wordpress://"
+}
+
+extension UIApplication {
```

Comment:
Since we only need this check to (temporarily) display the success card, how about we move it somewhere else than the `UIApplication` scope? It's not mandatory, but I kinda feel that the scope of this check is limited to My Site, so maybe we can move it there?
If, in the future, we feel that this should be moved up to a higher scope, we can always do so.

Reply:
👋 Hey @Gio2018 - @salimbraksa is working on the ["Success Card" PR](https://github.com/wordpress-mobile/WordPress-iOS/issues/19678) - this one if for detection of the WordPress app on a clean install of the Jetpack app. Will be glad to discuss more if you still think the scope is a factor here.

Reply:
I see. Again it's not necessary to change anything here. But everything feels still related to the migration context, rather than the whole app. But again you can safely ignore the comment, and maybe remind, as the change discussed above, to remove this once it's not needed anymore.

Reply:
I'll try to clarify my rationale behind the `AppScheme` suggestion but all suggestions sound good to me and I'll let you decide how you want to handle this. 🙇 

I understand this feels related to the migration context but I actually wasn't thinking from the migration perspective when I first thought about this. Instead, I thought of the built-in `UIApplication.canOpenURL(_: URL)` method, and I saw it was used before to check the availability of these apps:

1. `googlegmail://`
2. `tel://`
3. `mailto://`
4. `sms://`

But using the string-based version of these app schemes is not type-safe. So the idea was to store all those strings in a type like `AppScheme`. Then add the `UIApplication.canOpen(app: AppScheme)` method, which is just a convenient, type-safe version of `UIApplication.canOpenURL(_: URL)`.